### ISSUE TITLE
Many many errors in English spelling and punctuation

### ISSUE THREAD

**Issue by Its-The-Fuzz** [CrealityOfficial/CrealityPrint#162]
Creality Print 5 has many, many errors in spelling and punctuation throughout. If the Creality team wants to make the source file available, I would be happy to correct a bunch of these errors for you.

A very small number of examples:
The name of the application appears as 'Creality_Print' in many places, but I assume it's supposed to be 'Creality Print'
In paragraphs, there should always be a space after a period and before a new sentence
'OK' button label should be capitalized on Mac and Windows
Settings panel label capitalization is inconsistent
Menu command 'Save As Project' should be 'Save Project As…'
Menu command 'Recently files' should be 'Open Recent File'
Menu command 'Recently Opened Project' should be 'Open Recent Project'
'Upload to Crealitycloud' command should be 'Upload to Creality Cloud'

etc. etc. etc.

**Comment by raycardillo** [comment]
Yup. If it was open sourced by now we could start contributing fixes for many of these annoying little things. :D
